### ISSUE TITLE
[Footer]: QA: The <label> element does not have descriptive text that identifies the expected input

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8998]
### Description

Element code:
<label
    class="bx--label"
    part="label-text"
    for="input">

Issue : The <label> element does not have descriptive text that identifies the expected input

Expected: Add text to the empty <label> element that clearly identifies the purpose of the interactive component being labelled.

### Component(s) impacted

Footer

### Browser

Firefox

### Carbon for IBM.com version

v1.35.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
